src/git.rs

```diff
@@ -243,6 +244,31 @@ impl Git {
         Ok(())
     }
 
+    pub async fn fetch_from_remote(
+        &self,
+        refs: &[&GitHubBranch],
+        remote: &str,
+    ) -> Result<()> {
+        if !refs.is_empty() {
+            let mut command = async_process::Command::new("git");
+            command
+                .arg("fetch")
+                .arg("--no-write-fetch-head")
```

Comment:
note, in the future  we might want to avoid `--no-write-fetch-head` if the version of `git` is not new enough (as mentioned in https://github.com/getcord/spr/issues/61)

Reply:
:+1: